### PR TITLE
feat(format): script

### PR DESCRIPTION
**Pull request by barrett-ruth**
add format script for convenience
support eslint, prettier compatibility

### CODE REVIEW

**Comment by junholee6a** [comment]
The script does seem to check and fix the formatting for these files, but I get two errors when I run it. Are there more steps I need to take before running the format script?

```
junholee@Junhos-Air theCourseForum2-original % source scripts/format.sh
OCI runtime exec failed: exec failed: unable to start container process: exec: "black": executable file not found in $PATH: unknown
Skipped 2 files
.github/ISSUE_TEMPLATE/bug-report.md 27ms (unchanged)
.github/ISSUE_TEMPLATE/feature_request.md 6ms (unchanged)
.github/pull_request_template.md 2ms (unchanged)
docs/archive/grade-data.md 7ms (unchanged)
docs/archive/home.md 4ms (unchanged)
docs/archive/release-history.md 23ms (unchanged)
docs/archive/search.md 11ms (unchanged)
docs/archive/semester-data.md 6ms (unchanged)
docs/archive/update-guide.md 11ms (unchanged)
docs/dev.md 7ms (unchanged)
docs/useful-commands.md 7ms (unchanged)
README.md 6ms (unchanged)
tcf_website/static/about/about.css 16ms (unchanged)
tcf_website/static/about/history/history.css 5ms (unchanged)
tcf_website/static/base/base.css 10ms (unchanged)
tcf_website/static/base/bugform.js 31ms (unchanged)
tcf_website/static/base/messages.css 1ms (unchanged)
tcf_website/static/base/navbar.css 5ms (unchanged)
tcf_website/static/base/reset.css 12ms (unchanged)
tcf_website/static/base/sidebar.css 3ms (unchanged)
tcf_website/static/base/tcf-style.css 3ms (unchanged)
tcf_website/static/browse/browse.css 2ms (unchanged)
tcf_website/static/common/bind-show-hide.js 1ms (unchanged)
tcf_website/static/common/bind-sort.js 5ms (unchanged)
tcf_website/static/common/form.js 2ms (unchanged)
tcf_website/static/common/rating_card.css 1ms (unchanged)
tcf_website/static/common/show-hide.css 1ms (unchanged)
tcf_website/static/common/show-hide.js 2ms (unchanged)
tcf_website/static/common/sortClasses.js 14ms (unchanged)
tcf_website/static/common/sorting.js 6ms (unchanged)
tcf_website/static/course/course_professor.css 8ms (unchanged)
tcf_website/static/course/javascript/course_data.js 24ms (unchanged)
tcf_website/static/course/javascript/instr_data.js 2ms (unchanged)
tcf_website/static/department/department.css 1ms (unchanged)
tcf_website/static/department/loadMoreCourses.js 9ms (unchanged)
tcf_website/static/instructor/instructor.css 0ms (unchanged)
tcf_website/static/landing/dashboard.css 1ms (unchanged)
tcf_website/static/landing/feedbackform.js 2ms (unchanged)
tcf_website/static/landing/landing.css 5ms (unchanged)
tcf_website/static/login/login_button.css 1ms (unchanged)
tcf_website/static/profile/profile.css 0ms (unchanged)
tcf_website/static/qa/new_answer.js 4ms (unchanged)
tcf_website/static/qa/qa.css 1ms (unchanged)
tcf_website/static/qa/qa.js 9ms (unchanged)
tcf_website/static/qa/sort_qa.js 7ms (unchanged)
tcf_website/static/reviews/new_review.css 1ms (unchanged)
tcf_website/static/reviews/new_review.js 12ms (unchanged)
tcf_website/static/reviews/review_stats.css 1ms (unchanged)
tcf_website/static/reviews/review.css 2ms (unchanged)
tcf_website/static/reviews/review.js 4ms (unchanged)
tcf_website/static/reviews/sort_reviews.js 3ms (unchanged)
tcf_website/static/reviews/sync_review_sliders.js 2ms (unchanged)
tcf_website/static/search/search.css 1ms (unchanged)

Oops! Something went wrong! :(

ESLint: 7.9.0

ESLint couldn't find the plugin "eslint-plugin-prettier".

(The package "eslint-plugin-prettier" was not found when loaded as a Node module from the directory "/app/.config".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-prettier@latest --save-dev

The plugin "eslint-plugin-prettier" was referenced from the config file in "--config".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

**Comment by barrett-ruth** [comment]
yeha u need to rebuild the container from scratch bc it adds new dependencies. docker compose build --no-cache

File: scripts/format.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+docker exec tcf_django black --line-length 80 .
+docker exec tcf_django isort --profile black --line-length 80 .
+docker exec tcf_django npx prettier --write '**/*.{css,js,md}'
+docker exec tcf_django npx eslint --fix -c .config/.eslintrc.yml tcf_website/static/
```
Comment (junholee6a) [inline scripts/format.sh:6]: Still getting an error on this command, after rebuilding the docker container

```
Oops! Something went wrong! :(

ESLint: 7.9.0

ESLint couldn't find the plugin "eslint-plugin-prettier".

(The package "eslint-plugin-prettier" was not found when loaded as a Node module from the directory "/app/.config".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-prettier@latest --save-dev

The plugin "eslint-plugin-prettier" was referenced from the config file in "--config".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

**Comment by barrett-ruth** [comment]
@junholee6a

**Review by junholee6a — APPROVED** [review]
Works without errors!

**Pull request merged**